src/obis-observer/commands/uplink/GetContentByShortNameResponseStr.ts

```diff
@@ -0,0 +1,90 @@
+import Command, {TCommandExampleList} from '../../Command.js';
```

Comment:
there is a new `command size` field in docs

Reply:
added